go.mod

```diff
@@ -104,6 +104,7 @@ require (
 	github.com/modern-go/reflect2 v1.0.2 // indirect
 	github.com/monochromegane/go-gitignore v0.0.0-20200626010858-205db1a8cc00 // indirect
 	github.com/nxadm/tail v1.4.8 // indirect
+	github.com/olekukonko/tablewriter v0.0.5
```

Comment:
This new package is a library that is more feature rich than the library that we currently use. 
Also, this library is used in many other Kubernetes projects including k/k. Ref: https://grep.app/search?q=github.com/olekukonko/tablewriter&filter[repo.pattern][0]=kubernetes/

The existing table library usage can be completely replaced with this one so that we dont have to import 2 different table libraries. (Should be done in a separate PR.)

Reply:
opened https://github.com/kubernetes-sigs/cluster-api/issues/5941